pkg/metadata/inventories/inventories.go

```diff
@@ -108,6 +108,16 @@ const (
 	HostOSVersion AgentMetadataName = "os_version"
 )
 
+// Refresh signals that some data has been updated and a new payload should be sent (ex: when configuration is changed
```

Comment:
nit: We should break long lines at <100 chars. Same with the line below this one.

Reply:
I don't we ever enforce this. We have plenty of line above 80/100 chars (my current editor wrap at 120).

Reply:
> We have plenty of line above 80/100 chars 

I see your point but other code should not guide us on if we want to continue the practice with newly-added code. I only mentioned these few lines that stuck out as they look clunky even in GH UI for code review and I can only imagine that things would be worse if editing with default font size in a terminal window.

No need for me to bike-shed on this but maybe we should try to pick some sort of a limit for our editors so that we keep things consistent.